optimum/onnxruntime/configuration.py

```diff
@@ -604,73 +608,222 @@ def tensorrt(
 class OptimizationConfig:
     """
     OptimizationConfig is the configuration class handling all the ONNX Runtime optimization parameters.
+    There are two stacks of optimizations:
+        1. The ONNX Runtime general-purpose optimization tool: it can work on any ONNX model.
+        2. The ONNX Runtime transformers optimization tool: it can only work on a subset of transformers models.
 
-    Args:
-        optimization_level (`int`, defaults to 1):
-            ONNX opset version to export the model with.
+    Attributes:
+        general_optimization_level (`int`, defaults to 1):
```

Comment:
Not a big fan of this name, but basically trying to convey that this `optimization_level` is about the general ONNX Runtime tool, and not the `transformers`-specific optimizations.

Do you have any better suggestion @echarlaix @regisss ?

Reply:
I think we should keep `optimization_level` here because I have the feeling more accurate names will confuse users. This is not great but I don't think there is a perfect solution.
That means that we need to make it very clear in the documentation.